joebvp/VPmeasure.py

```diff
@@ -606,15 +600,38 @@ def batch_fit(spec,filelist,outparfile=None,outmodelfile=None,**kwargs):
             q_pass += 1
         except:
             print('VPmeasure: Fitting failed:',ff)
+            fails += [ff]
             q_fail += 1
     print("")
     print("VPmeasure: {}/{} fits converged, {}/{} failed (see log for details).\n".format(q_pass, i, q_fail, i))
 
+    # ask whether run GUI mode on failures
+    if q_fail >= 0:
+        print("VPmeasure: Failed runs are: {}\n".format(fails))
+        while True:
```

Comment:
Not sure I follow the logic here.  What is 'True' exactly?

Reply:
`True` is `True` ;) This means that will loop forever until something breaks it. In this case, only answers `in ['y','n', 'yes','no']` will break the loop. There could be a better way to do this, but this was simple enough.